cpp/installer.cc

```diff
@@ -552,6 +552,12 @@ void InstallUBTools(const bool make_install, const OSSystemType os_system_type)
         FileUtil::MakeDirectoryOrDie(UBTools::GetTueLocalTmpPath(), /* recursive = */true);
     }
 
+    // ...then create /var/www/custom_rss_feeds
+    if (not FileUtil::Exists(UBTools::GetTuelibPath())) {
+        Echo("creating /var/www/custom_rss_feeds");
```

Comment:
I don't think we should put the feeds into a new subdir in /var/www.

As we discussed so far, there will be a VuFind Controller which calls the program every time a feed needs to be generated. The Controller could either take the target path as an argument (which would be more likely somewhere in /tmp and the VuFind Controller will then deliver it), or write it directly to stdout.

Reply:
We now write the XML output to /dev/stdout.